chain/README.md

```diff
@@ -25,6 +25,12 @@ cargo install commonware-deployer
 cargo run --bin setup -- --peers 10 --bootstrappers 2 --regions us-west-1,us-east-1,eu-west-1,ap-northeast-1,eu-north-1,ap-south-1,sa-east-1,eu-central-1,ap-northeast-2,ap-southeast-2 --instance-type c7g.xlarge --storage-size 10 --storage-class gp3 --worker-threads 4 --message-backlog 16384 --mailbox-size 16384 --dashboard dashboard.json --output assets
 ```
 
+#### [Optional] Upload Consensus Artifacts to Indexer
+
+```bash
+--indexer <indexer URL>
```

Comment:
TODO: Add this as an "apply" command that reads existing configs and inserts (usually want to use the "identity" from above when deploying said indexer).